service/src/main/resources/config/aou/test/SC2022Q4R6/original/entity/t_brand.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name" : "t_brand",
+  "idAttribute" : "id",
+  "attributes" : [ {
+    "type" : "KEY_AND_DISPLAY",
+    "name" : "standard_concept",
+    "dataType" : "STRING",
+    "displayHint" : {
```

Comment:
I wouldn't expect the `original` files to include display hints, which are generated by the `EXPAND_CONFIG` step. Did you maybe overwrite the original version by mistake? Ditto on some of the other files (e.g. `t_ingredient`) and in the SR underlay.

Reply:
This must be a copy, but let me check and remove the displayHint tags